index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
```

Comment:
Suggest changing "Game over, refresh to play again" to "Game Over, Refresh to restart"